spatialdb/object.py

```diff
@@ -293,6 +297,19 @@ def generate_object_key(self, resource, resolution, time_sample, morton_id, iso=
 
         return "{}&{}".format(hash_str, base_key)
 
+    @staticmethod
+    def generate_lookup_key(collection_id, experiment_id, channel_id, resolution):
+        """
+        Generate the lookup key for storing as an attribute in the S3 index table.
+
+        This value will be the key of a global secondary index that allows
+        finding all the cuboids belonging to a particular channel in an 
+        efficient manner.
+        """
+        lookup_key = '{}&{}&{}&{}'.format(
+            collection_id, experiment_id, channel_id, resolution)
```

Comment:
ToDo: append random number between 1 - N so we can spread out the writes during ingests to avoid throttling.